addons/account/tests/common.py

```diff
@@ -92,6 +90,13 @@ def setUpClass(cls, chart_template_ref=None):
             'standard_price': 160.0,
             'property_account_income_id': cls.copy_account(cls.company_data['default_account_revenue']).id,
             'property_account_expense_id': cls.copy_account(cls.company_data['default_account_expense']).id,
+        })
+        cls.env['product.template'].invalidate_model()
+        cls.product_a.write({
+            'taxes_id': [Command.set(cls.tax_sale_a.ids)],
+            'supplier_taxes_id': [Command.set(cls.tax_purchase_a.ids)],
+        })
+        cls.product_b.write({
```

Comment:
any reason why you do that in write() instead of directly during the create(), few lines above? :thinking: